lib/ecto/adapters/postgres/connection.ex

```diff
@@ -171,10 +171,9 @@ if Code.ensure_loaded?(Postgrex.Connection) do
 
     defp join([], _sources), do: nil
     defp join(joins, sources) do
```

Comment:
@jquadrin you will need to add to chose changes to your pull request once I merge this to master and you rebase.